javascript.js

```diff
@@ -51,10 +51,69 @@ getFunctionAdd.addEventListener("click", function(){
 
 function removeElement(parameter) {
   if(parameter != "No") {
-    console.log("getNumberArticle", parameter);
     let transform = document.getElementsByClassName(parameter)[0];
     transform.remove();
   }
 }
```

Comment:
- [ ] Kindly make sure that the remove button works as intended and remove books from the list and the local storage.

Reply:
Now it is removed from local and from the UI, Thanks